proto/viam/app/mltraining/v1/ml_training.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package viam.app.mltraining.v1;
+
+import "app/data/v1/data.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/mltraining/v1";
+
+service MLTrainingService {
+  rpc SubmitTrainingJob(SubmitTrainingJobRequest) returns (SubmitTrainingJobResponse);
+  rpc GetTrainingJob(GetTrainingJobRequest) returns (GetTrainingJobResponse);
+}
+
+enum ModelType {
+  MODEL_TYPE_UNSPECIFIED = 0;
+  MODEL_TYPE_MULTI_CLASS_CLASSIFICATION = 1;
```

Comment:
After talking this through with design, I realize this should be SINGLE_LABEL_CLASSIFICATION and MULTI_LABEL_CLASSIFICATION. Multi-class inherently means > 1 classes, whereas I was assuming it was 1+ classes. See the breakdown [here](https://cloud.google.com/vertex-ai/docs/image-data/classification/prepare-data)

For either single- or multi-label, there could be multiple possible labels as inputs (dog, cat) but the above distinctions determines whether we'd only predict one (dog or cat) or multiple (dog and cat) per image.
For the binary classification case, the user would only pick one input label (dog) and single label.

Will update accordingly in the scope if have agreement here, cc @tahiyasalam

Reply:
Sounds good, changed

Reply:
@tahiyasalam just double checking - do you agree?

Reply:
Oops, sorry missed the sentence about agreement and thought you were just cc'ing me.

But yes, I agree!